docs/reference/docs/reindex.asciidoc

```diff
@@ -4,19 +4,24 @@
 <titleabbrev>Reindex</titleabbrev>
 ++++
 
+<<<<<<< HEAD
```

Comment:
Looks like some git conflict markers are still in here.

Reply:
Good catch @gtback 👍 I resolved those issues.